examples/skeleton/basicfwd.c

```diff
@@ -102,7 +102,7 @@ port_init(uint16_t port, struct rte_mempool *mbuf_pool)
 	if (retval != 0)
 		return retval;
 
-	return 0;
+	return retval;
```

Comment:
At line 100 retval is set to zero or some value, but then at line 102 if retval != 0 we return retval, which means line 105 return 0 is ok too.

Reply:
The code itself has no issues , but i feel returning a variable made the code more readable...

Reply:
I understand, but I believe the DPDK community will not see it that way IMO. As the only value that can be returned in that spot is zero :-(

Reply:
Sure thing, Thanks for getting back on this!